www/js/diary/infinite_scroll_place_item.js

```diff
@@ -0,0 +1,47 @@
+
+/**
+ * A directive to display each place within the diary view.
+ */
+
+angular.module('emission.main.diary.infscrollplaceitem',
+    ['emission.main.diary.infscrolllist',
+        'emission.survey.multilabel.services',
+        'emission.main.diary.infscrolldetail',
+        'ng-walkthrough',
+        'nvd3',
+        'emission.plugin.kvstore',
+        'emission.services',
+        'emission.config.imperial',
+        'emission.config.dynamic',
+        'emission.plugin.logger',
+        'emission.stats.clientstats',
+        'emission.survey.enketo.add-note-button',])
+
+  .directive("infiniteScrollPlaceItem", function () {
+    return {
+      restrict: 'E',
+      scope: {
+        trip: '=',
+        config: '=',
+        surveys: '=',
+      },
+      controller: 'PlaceItemCtrl',
+      templateUrl: 'templates/diary/place_list_item.html'
+    };
+  })
+
+  .controller("PlaceItemCtrl", function ($scope, $state, DynamicConfig) {
+    console.log("Place Item Controller called");
+    console.log('config is ', $scope.config);
+
+    $scope.timeBounds = () => {
+      let obj = {
+        isPlace: true,
+        start_fmt_time: $scope.trip?.end_fmt_time,
+        end_fmt_time: $scope.trip?.nextTrip?.start_fmt_time
+      }
```

Comment:
I found that for this change, `isPlace` is actually a useful variable https://github.com/e-mission/e-mission-phone/pull/917/commits/23be4a8b75b7d069a64b49add0c6be3077344058

Reply:
@sebastianbarry this is primarily polish at this point, so feel free to resolve this again, but theoretically, you don't need `isPlace`.

wrt "- split the one previous "start/stop time" lines into one that checks isplace, then uses enter/exit, otherwise uses start/stop", you can simply use enter/exit if they exist and start/stop if they don't

Reply:
Confirmed this was changed in commit https://github.com/sebastianbarry/e-mission-phone/commit/23be4a8b75b7d069a64b49add0c6be3077344058 to 
```javascript
        enter_fmt_time: $scope.trip?.end_fmt_time,
        exit_fmt_time: $scope.trip?.nextTrip?.start_fmt_time
      }
```

This is good to resolve

Reply:
when we switch to confirmed places, you should use the `enter_fmt_time` and `exit_fmt_time` from the place instead. you can resolve this comment after reading this message.